src/Compilers/CSharp/Portable/Symbols/PropertySymbolExtensions.cs

```diff
@@ -63,7 +63,7 @@ public static bool HasRefOrOutParameter(this PropertySymbol property)
         {
             foreach (ParameterSymbol param in property.Parameters)
             {
-                if (param.RefKind != RefKind.None)
+                if (param.RefKind == RefKind.Ref || param.RefKind == RefKind.Out)
```

Comment:
File `IsConstModifierTests.cs` in C# emit tests project. You can look for `IsConstModReqIsConsumedInRefCustomModifiersPosition_IL_Indexers_ReturnTypes` and `IsConstModReqIsConsumedInRefCustomModifiersPosition_Code_Indexers_ReturnTypes` now.

---
In reply to: [119789809](https://github.com/dotnet/roslyn/pull/19658#discussion_r119789809) [](ancestors = 119789809,118609191,118061963)